src/natcap/invest/datastack.py

```diff
@@ -311,77 +192,217 @@ def build_datastack_archive(args, model_name, datastack_path):
     Returns:
         ``None``
     """
+    module = importlib.import_module(name=model_name)
+
+    # Allow the model to override the common datastack function.  This is
+    # useful for tables (like HRA) that are too complicated to describe in the
+    # ARGS_SPEC format.
+    if hasattr(module, 'build_datastack_archive'):
+        return module.build_datastack_archive(args, datastack_path)
+
     args = args.copy()
     temp_workspace = tempfile.mkdtemp(prefix='datastack_')
-    logfile = os.path.join(temp_workspace, 'log')
     data_dir = os.path.join(temp_workspace, 'data')
     os.makedirs(data_dir)
 
-    # For tracking existing files so we don't copy things twice
+    # write a logfile to the archive
+    logfile = os.path.join(temp_workspace, 'log')
```

Comment:
This comes down to personal preference I guess, but what about `log.txt`?

Reply:
I don't see why not!  07e827c84